terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 1: 27925 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 2: 27927 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 3: 27929 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 4: 27931 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 5: 27933 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 6: 27935 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 7: 27937 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 8: 27939 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 9: 27941 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 10: 27943 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 11: 27945 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 12: 27947 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 13: 27949 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 14: 27951 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 15: 27953 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 16: 27955 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 17: 27957 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 18: 27959 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 19: 27961 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 20: 27963 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 21: 27965 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 22: 27967 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 23: 27969 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 24: 27971 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 25: 27973 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 26: 27975 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 27: 27977 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 28: 27979 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 29: 27981 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 30: 27983 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 31: 27985 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 32: 27987 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 33: 27989 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 34: 27991 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 35: 27993 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 36: 27995 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 37: 27997 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 38: 27999 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 39: 28001 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 40: 28003 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 41: 28005 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 42: 28007 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 43: 28009 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 44: 28011 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 45: 28014 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 46: 28016 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 47: 28018 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 48: 28020 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 49: 28022 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 50: 28024 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 51: 28026 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 52: 28028 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 53: 28030 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 54: 28032 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 55: 28034 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 56: 28036 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 57: 28038 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 58: 28040 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 59: 28042 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 60: 28044 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 61: 28046 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 62: 28048 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 63: 28050 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 64: 28052 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 65: 28054 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 66: 28056 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 67: 28058 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 68: 28060 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 69: 28062 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 70: 28064 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 71: 28066 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 72: 28068 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 73: 28070 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 74: 28072 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 75: 28074 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 76: 28076 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 77: 28078 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 78: 28080 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 79: 28082 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 80: 28084 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 81: 28086 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 82: 28088 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 83: 28090 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 84: 28092 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 85: 28094 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 86: 28096 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-pos-a.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 87: 28098 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-a.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 88: 28100 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 89: 28102 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 90: 28104 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 91: 28106 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-c.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 92: 28108 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 93: 28110 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 94: 28112 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 95: 28114 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 96: 28116 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-b.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 97: 28118 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 98: 28120 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 99: 28122 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/37.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-3-b-val.sh: line 100: 28124 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-c.pddl $1
